docker-compose.yml

```diff
@@ -1,7 +1,7 @@
 # -*- coding: utf-8 -*-
 #
 # This file is part of CERN Open Data Portal.
-# Copyright (C) 2015, 2016, 2017 CERN.
+# Copyright (C) 2015, 2016, 2017, 2020 CERN.
```

Comment:
CSRF validation will be performed by default, are we affected?
```console
web_1            | /opt/rh/rh-python36/root/usr/lib/python3.6/site-packages/invenio_rest/ext.py:30: FutureWarning: CSRF validation will be enabled by default in the version 1.3.x
web_1            |   self.init_app(app)
```

Reply:
IIRC it should be OK.